tests/unit/libbpf_test.cpp

```diff
@@ -34,6 +34,8 @@
     TEST_CASE(CONCAT(_name, "-jit"), _group) { _function(EBPF_EXECUTION_JIT); } \
     TEST_CASE(CONCAT(_name, "-native"), _group) { _function(EBPF_EXECUTION_NATIVE); }
 
+const int nonexistant_fd = 12345678;
```

Comment:
```suggestion
const int nonexistent_fd = 12345678;
```
typo is still not fixed

Reply:
Fixed other half of typo.